modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,62 +1,68 @@
 @import 'atlas-variables';
+@import 'cadmin-variables';
 
-.lfr-analytics-reports-panel {
-	&.open-admin-panel.sidenav-menu-slider {
-		visibility: visible;
-		width: 330px;
-	}
+html#{$cadmin-selector} {
+	.cadmin {
+		&.lfr-analytics-reports-panel {
+			&.open-admin-panel.sidenav-menu-slider {
+				visibility: visible;
+				width: 320px;
+			}
 
-	.sidebar {
-		border-left: 1px solid $gray-300;
-		box-shadow: none;
-	}
+			.sidebar {
+				border-left: 1px solid $gray-300;
```

Comment:
We need to prefix all variables with `$cadmin-`, so this one should be `$cadmin-gray-300`.